react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function Pokemon() {
+  const pokedex = [
+    { number: '001', name: 'Bulbasaur' },
+    { number: '004', name: 'Charmander' },
+    { number: '007', name: 'Squirtle' },
+    { number: '025', name: 'Pikachu' },
+    { number: '039', name: 'Jigglypuff' }
+  ];
```

Comment:
`pokedex` should be defined in the global scope and passed into the `Pokemon` component as a `prop`.